extensions/api/data-management/catalog/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/catalog/service/CatalogService.java

```diff
@@ -0,0 +1,29 @@
+/*
+ *  Copyright (c) 2020 - 2022 Bayerische Motoren Werke Aktiengesellschaft
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.api.datamanagement.catalog.service;
+
+import org.eclipse.dataspaceconnector.spi.types.domain.catalog.Catalog;
+
+import java.util.concurrent.CompletableFuture;
+
+public interface CatalogService {
```

Comment:
There is already a `CatalogService` interface. Shouldnt those be merged?

Reply:
I'm not sure it would be a good idea, they serve different purposes and have different dependencies

Reply:
Since we decided to have the `*Service` notation on the data management api, probably the other could have another name, like `CatalogProvider` or `CatalogGenerator`